plaidml/bridge/openvino/src/plaidml_plugin/plaidml_util.hpp

```diff
@@ -19,7 +20,8 @@
 
 namespace PlaidMLPlugin {
 
-ngraph::AxisSet get_axes_from_constant_operand(size_t operand_idx, ngraph::Node* layer);
+ngraph::AxisSet get_axes_set_from_constant_operand(size_t operand_idx, ngraph::Node* layer);
+ngraph::AxisVector get_axes_vector_from_constant_operand(size_t operand_idx, ngraph::Node* layer);
```

Comment:
I think we should rename to:

`get_axis_set` and `get_axis_vector`, because to me, `axes` is the plural of `axis` and a set or a vector is valid plurality of `axis`. So `get_axes_set` makes it feel like it should be returning a `vector<AxisSet>`.